package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
-    "prettier": "^2.2.1"
+    "prettier": "^2.2.1",
```

Comment:
Also is there a reason for those new packages to be added to the main package.json? If not this should be removed as well as the change to the package-lock.json